soh/soh/Enhancements/randomizer/3drando/menu.cpp

```diff
@@ -531,7 +531,8 @@ std::string GenerateRandomizer(std::unordered_map<RandomizerSettingKey, uint8_t>
         return "";
     } else {
         try {
-            int seed = std::stoi(seedInput, nullptr);
+            //int seed = std::stoi(seedInput, nullptr);
+            int seed = std::stoul(seedInput, nullptr) & 0xFFFFFFFF;
```

Comment:
Can get rid of the commented line.

Regarding the `stoul` being truncated with `0xFFFFFFFF`, this should be fine since `seedInput` should have already been truncated to a `uint32_t` value range by the randomizer menu.